src/main/java/com/metamx/common/parsers/JSONPathParser.java

```diff
@@ -60,7 +59,6 @@
    */
   public JSONPathParser(List<FieldSpec> fieldSpecs, boolean useFieldDiscovery, ObjectMapper mapper)
   {
-    this.fieldSpecs = fieldSpecs;
```

Comment:
If this is not used, can we `@Deprecated` this constructor signature and make one that does not have `fieldSpecs`?